plugins/chanbackup.c

```diff
@@ -742,7 +742,7 @@ int main(int argc, char *argv[])
         set_feature_bit(&features->bits[INIT_FEATURE],
                         YOUR_PEER_STORAGE_FEATUREBIT);
 
-	plugin_main(argv, init, PLUGIN_RESTARTABLE, true, NULL,
+	plugin_main(argv, init, PLUGIN_STATIC, true, NULL,
```

Comment:
I guess this is arguable?  It's not terrible to restart it, for example.  It still works...

Reply:
`Custom featurebits only allows for non-dynamic.`